relay-event-schema/src/protocol/span.rs

```diff
@@ -315,6 +315,18 @@ pub struct SpanData {
     #[metastructure(field = "sentry.frames.total", legacy_alias = "frames.total")]
     pub frames_total: Annotated<Value>,
 
+    /// Message Retry Count
+    #[metastructure(field = "sentry.messaging.message.retry.count")]
+    pub messaging_message_retry_count: Annotated<Value>,
+
+    /// Message Receive Latency
+    #[metastructure(field = "sentry.messaging.message.receive.latency")]
+    pub messaging_message_receive_latency: Annotated<Value>,
+
+    /// Message Body Size
+    #[metastructure(field = "sentry.messaging.message.body.size")]
+    pub messaging_message_body_size: Annotated<Value>,
```

Comment:
These are new fields being added to the SDKs! I'll double check with the SDK team to see what they think

Reply:
About sending it as metrics: Should the SDK start sending metrics with an (maybe auto enabled) integration? I think this could make users mad because metrics have to be paid separately. What is our consensus about this?

Another question: should we in general add "string" data in spans as `span.data` and "number" data as `measurements`?

Reply:
> About sending it as metrics: Should the SDK start sending metrics with an (maybe auto enabled) integration? I think this could make users mad because metrics have to be paid separately. What is our consensus about this?

We don't want to launch the queues module with an added performance cost at the moment, so I think we'll have to rule out sending metrics if extra cost is involved. (cc @bcoe)